rocAL/include/api/rocal_api_data_loaders.h

```diff
@@ -56,7 +56,8 @@ extern "C" RocalTensor ROCAL_API_CALL rocalJpegFileSource(RocalContext context,
                                                           bool shuffle = false,
                                                           bool loop = false,
                                                           RocalImageSizeEvaluationPolicy decode_size_policy = ROCAL_USE_MOST_FREQUENT_SIZE,
-                                                          unsigned max_width = 0, unsigned max_height = 0, RocalDecoderType rocal_decoder_type = RocalDecoderType::ROCAL_DECODER_TJPEG, std::pair<RocalLastBatchPolicy, bool> last_batch_info = {RocalLastBatchPolicy::ROCAL_LAST_BATCH_FILL, true});
+                                                          unsigned max_width = 0, unsigned max_height = 0, RocalDecoderType rocal_decoder_type = RocalDecoderType::ROCAL_DECODER_TJPEG,
+                                                          ShardingInfo sharding_info = {});
```

Comment:
initialize to default instead, sharding_info = default;
change all such instances

Reply:
We will not be able to use the default keyword directly, hence using ShardingInfo() to do the same.